chore: prepare release 0.17.0 of HAL Explorer WebJar

toedter opened the pull request.

toedter committed:
Merge pull request #2 from webjars/master

chore: adapt James' workflow and versioning

toedter committed:
chore: update to version 0.17.1-SNAPSHOT

This will release the 0.17.0 WebJar

jamesward commented on pom.xml:
```diff
@@ -11,7 +11,7 @@
     <groupId>org.webjars</groupId>
     <artifactId>hal-explorer</artifactId>
-    <version>0.16.1-SNAPSHOT</version>
+    <version>0.17.1-SNAPSHOT</version>
```
```suggestion
    <version>0.17.0-SNAPSHOT</version>
```

toedter committed:
Update pom.xml

Hi James, I thought that your "unsnapshot" automatically goes from 0.17.1-SNAPSHOT to 0.17.0

Co-authored-by: James Ward <james@jamesward.com>
